tests/test_validator.py

```diff
@@ -139,7 +139,7 @@ def test_invalid_submission_file(self):
         # Exit correctly with code 0
         self.assertEqual(result.exit_code, 0)
         # The correct output is given
-        self.assertIn("The XML file: invalid_SUBMISSION.xml\nis invalid.",
+        self.assertIn("The XML file: invalid_SUBMISSION.xml\nis invalid.\n\n",
```

Comment:
This could now be `assertEqual` then but that would also make the line 81 characters long :D

Reply:
done. moved it to a var to preserve char line length